### DISCUSSION TITLE
influxdb password not in sync with SOC password after change

### DISCUSSION THREAD

**Discussion by ponte19** [Security-Onion-Solutions/securityonion#11562] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### CPU

32

### RAM

128 GB

### Storage for /

3 TB

### Storage for /nsm

1 TB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I figured out that when you change the SOC password in the web-browser it will not sync with influxdb. I can only sign in with the first ever password.

I tried sudo salt-call state.highstate and a reboot, too. The influxdb password is still my first ever password.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by InfosecGoon** [comment, marked as answer]
How are you resetting the password? If you do it through the Profile icon in the upper-left corner of the screen, it will not reset InfluxDB - there's a warning banner there that says as much. However, if you reset through Administration --> Users --> Access Control under your account, it should reset everything.

**Reply by ponte19** [reply]
Thank you for the advice. I just reset my password in the admin panel and I am waiting for the sync-process to take place. Unfortunatelly my instance of Security Onion does not tell me about being not able to login in influx anymore. It just tells me that I need to re-login but it does not tell me that I am simply not being able to login anymore.

To my mind this is unnecessary strangeness when there are two password-change functions but only one is working like it is supposed to be.
